packages/react-router-dom/htmlescape.ts

```diff
@@ -0,0 +1,16 @@
+// This code is copy form https://github.com/vercel/next.js
+// License: https://github.com/vercel/next.js/blob/b2e9431bb46563264d450b1707dc0d9cdaf70d26/license.md
```

Comment:
```suggestion
// This utility is based on https://github.com/zertosh/htmlescape
// License: https://github.com/zertosh/htmlescape/blob/0527ca7156a524d256101bb310a9f970f63078ad/LICENSE
```

Since this file is already copied by Next.js, let's reference the original code as well.